apps/dot-voting/contracts/lib/ADynamicForwarder.sol

```diff
@@ -8,9 +8,21 @@ import "@aragon/os/contracts/lib/math/SafeMath.sol";
 import "@aragon/os/contracts/lib/math/SafeMath64.sol";
 
 // TODO: Use @aragon/os/contracts/ version when it gets merged
-import "../evmscript/DynamicScriptHelpers.sol";
+import "./DynamicScriptHelpers.sol";
 // TODO: Research why using the @aragon/os version breaks coverage
-import "@aragon/os/contracts/common/IForwarder.sol";
```

Comment:
Let's not re-implement this until we decide how to proceed with renaming this import during coverage

Reply:
AFAIK, there is no coverage bug, I properly tested the change and coverage is working.

In my opinion this is the best tradeoff we can opt for now, as there was no other approach suggested for months.

We should not invest more time thinking on this or keep maintaining the ridiculous `tps/test-helpers` module just for this reason, I think we should instead focus into upgrading `@aragon/os` upstream to import the ADynamicForwarder from there, so as temporary solution, it seems fine for me.

What is your specific concern about this solution @topocount did you have the chance to test this new code to check that coverage is working correctly?

Now is the moment to decide, this task was around for months and we are always postposing it, I spent a reasonable amount of time testing that coverage works correctly, so, please, let's decide on that, please, be more specific and share your concerns about the change so we can address them.